package.json

```diff
@@ -23,14 +23,20 @@
     ]
   },
   "scripts": {
-    "test": "./node_modules/.bin/karma start --single-run"
+    "test": "karma start --single-run",
+    "grunt:dev": "grunt dev",
+    "grunt:rel": "grunt rel",
+    "grunt:watch": "grunt watch",
+    "grunt:lint": "grunt lint",
+    "develop": "npm-run-all grunt:dev grunt:watch"
```

Comment:
I've used `npm-run-all` in several projects. It makes it easy to run multiple npm scripts in parallel or serially across environments.

In this example it will first run `grunt:dev`, and then  `grunt:watch`.

Info about npm-run-all: https://github.com/mysticatea/npm-run-all/blob/master/docs/npm-run-all.md